contracts/MyMultiSig.cdc

```diff
@@ -252,6 +151,86 @@ pub contract MyMultiSigV2 {
+                    "This address has already signed."
+            }
+            let action = self.borrowAction(actionUUID:actionUUID)
+
+            // Validate Message
+            assert(
+                MyMultiSigV2.approveOrRevokeActionMessageIsValid(action: action, messageSignaturePayload: messageSignaturePayload),
+                message: "Signed message is invalid"
+            )
+        
+            // Validate Signature
+            var signatureValidationResponse = MyMultiSigV2.validateSignature(payload: messageSignaturePayload)
+            assert(signatureValidationResponse.isValid == true, message: "Invalid Signatures")
+            assert(signatureValidationResponse.totalWeight >= 999.0, message: "Total weight of combined signatures did not satisfy 999 key weight requirement.")
+
+            // Approve action
+            action.setAccountsVerified(signer: messageSignaturePayload.signingAddr, value: true)
+
+            emit ActionApprovedBySigner(address: messageSignaturePayload.signingAddr, uuid: self.uuid)
+            return signatureValidationResponse.isValid
+        }
+
+        pub fun signerRejectAction(actionUUID: UInt64, messageSignaturePayload: MessageSignaturePayload): Bool {
+            pre {
+                self.actions[actionUUID] != nil: "Couldn't find action with UUID ".concat(actionUUID.toString())
+                self.borrowAction(actionUUID: actionUUID).accountsVerified[messageSignaturePayload.signingAddr] != false:
+                    "Signer "
+                        .concat(messageSignaturePayload.signingAddr.toString())
+                        .concat(" has already rejected this action")
+            }
+
+            // Validate Message
+            assert(
+                MyMultiSigV2.approveOrRevokeActionMessageIsValid(action: self.borrowAction(actionUUID: actionUUID), messageSignaturePayload: messageSignaturePayload),
+                message: "Signed message is invalid"
+            )
+        
+            // Validate Signature
+            var signatureValidationResponse = MyMultiSigV2.validateSignature(payload: messageSignaturePayload)
+            assert(signatureValidationResponse.isValid == true, message: "Invalid Signatures")
+            assert(signatureValidationResponse.totalWeight >= 999.0, message: "Total weight of combined signatures did not satisfy 999 key weight requirement.")
+
+
+            let action = self.borrowAction(actionUUID: actionUUID)
+
+
+            // Revoke approval
+            action.setAccountsVerified(signer: messageSignaturePayload.signingAddr, value: false)
+
+            emit ActionRejectedBySigner(address: messageSignaturePayload.signingAddr, uuid: self.uuid)
+
+            // Check if we have enough rejections to delete the action
+            if  self.getTotalRejectedForAction(actionUUID: actionUUID) > (self.signers.length - Int(self.threshold)) {
```

Comment:
nit: brackets in the if clause

Reply:
@KsenijaZivkovic are you saying that it should be formatted like this?
```
if something
{
// stuff
}
```

Reply:
i think i prefer the open bracket on the same line as the `if` clause, personally